qemu/tests/virtual_nic_stress.py

```diff
@@ -92,11 +90,12 @@ def unload_stress(session):
                           " Check if the network is still alive",
                           logging.info)
     count = params["count"]
-    timeout = float(count) * 2
-    status, output = utils_net.ping(host_ip, count, session=session,
```

Comment:
Same here.

Reply:
Hi @PaulYuuu Please help me review it again.Thanks.